CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# 3.0.0
+- Release for dart sdk `>=2.13` (Support for packed structs and inline arrays)
```

Comment:
Period at end of sentence.

Reply:
Done. For once, can I not make this mistake! :p